warp-core/src/main/scala/com/workday/warp/utils/AnnotationReader.scala

```diff
@@ -39,9 +40,9 @@ object AnnotationReader extends StackTraceFilter {
    */
   protected def getWarpTestMethod(testId: String): Option[Method] = {
     val methodName: String = testId drop testId.lastIndexOf('.') + 1
-    // we expect the reflected method to have 0 parameters
-    val parameterTypes: Array[Class[_]] = Array.empty
-    this.getWarpTestClass(testId) flatMap { clazz => Try(Option(clazz.getMethod(methodName, parameterTypes: _*))).toOption.flatten }
+    // TODO using find won't work correctly wrt method overloading, its possible that we will have
+    // multiple junit test methods with the same name and we can't disambiguate
+    this.getWarpTestClass(testId).flatMap(_.getMethods.find(_.getName == methodName))
```

Comment:
if we find multiple methods do we want to output a warning so it's easier to track down that there were multiple methods with the same name?

Reply:
good idea i'll add that